lib/AppInfo/Application.php

```diff
@@ -227,6 +230,10 @@ public function register(IRegistrationContext $context): void {
 		$context->registerEventListener(MessageParseEvent::class, SystemMessage::class, 9999);
 		$context->registerEventListener(MessageParseEvent::class, UserMention::class, -100);
 
+		// Calendar listeners
+		$context->registerEventListener(CalendarObjectCreatedEvent::class, CalDavEventListener::class);
```

Comment:
psalm is not happy, we should add a stub in tests/stubs similar to https://github.com/nextcloud/spreed/blob/main/tests/stubs/oca_files_events.php